modin/core/storage_formats/base/query_compiler.py

```diff
@@ -4766,4 +4767,36 @@ def compare(self, other, align_axis, keep_shape, keep_equal, result_names):
             result_names=result_names,
         )
 
+    def repartition(self, axis):
```

Comment:
```suggestion
    def repartition(self, axis=None):
```
It seems to me strange if we pass None as a required arg.